### PR TITLE
fix(plugin-node-surrounding-code): Ensure surrounding code is truncated to sensible length

### PR DESCRIPTION
**Pull request by bengourley**
For minified code a single line can be unreasonably long, causing the payload to be rejected because
it's over the 1MB size limit. This change just takes the first 200 characters from the line.

### CODE REVIEW

File: packages/plugin-node-surrounding-code/surrounding-code.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ class CodeRange extends Writable {
     this._n++
     if (this._n < this._start) return cb(null)
     if (this._n <= this._end) {
-      this._code[String(this._n)] = chunk
```
Comment (snmaynard) [inline packages/plugin-node-surrounding-code/surrounding-code.js]: Is the substring centered on the column provided if we get one or is it just the start of line?

Comment (bengourley) [reply packages/plugin-node-surrounding-code/surrounding-code.js]: It's just the start of the line. We could center around the column but only for one of the lines.

Comment (snmaynard) [reply packages/plugin-node-surrounding-code/surrounding-code.js]: yeah - might get confusing. ok!

File: packages/plugin-node-surrounding-code/test/surrounding-code.test.js [hunk 2]
```diff
@@ -207,4 +207,32 @@ describe('plugin: node surrounding code', () => {
       }
     ]))
   })
+
+  it('truncates lines to 150 characters', done => {
```
Comment (fractalwrench) [inline packages/plugin-node-surrounding-code/test/surrounding-code.test.js]: should this say 200 characters?

Comment (bengourley) [reply packages/plugin-node-surrounding-code/test/surrounding-code.test.js]: yes – or better still only have the constant in the assertion and make the scenario description generic

**Pull request merged**